examples/simple_ssr/README.md

```diff
@@ -0,0 +1,6 @@
+# Server-side Rendering Example
```

Comment:
You forgot to add your example in the table in `examples/README.md`
But don't add it now, I'm changing that table so ill just add it in my PR.